content/body/video-content.php

```diff
@@ -50,19 +50,19 @@
 </p>
 
 <p>What makes a video accessible is widely misunderstood. Many web professionals know about closed captions.
-  What many don't know is that they absolutely need audio descriptions in order to be WCAG AA compliant.</p>
+  What many don't know is that they need audio descriptions to be WCAG AA compliant.</p>
```

Comment:
It's not the audio descriptions themselves that are to be WCAG AA compliant:

```suggestion
  What many don't know is that they need audio descriptions in order to be WCAG AA compliant.</p>
```